anotherfile.js

```diff
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
+    case "add":
+      return a + b;
+    case "subtract":
+      return a - b;
+    case "multiply":
+      return a * b;
+    case "divide":
+if (b === 0) throw new Error("Cannot divide by zero");
+      return a / b;
+    default:
+      return "Error: Unknown operation";
```

Comment:
Throw an error for unknown operations to maintain consistent error handling.

```suggestion
function calculate(operation, a, b) {
  try {
    switch (operation) {
      case "add":
        return a + b;
      case "subtract":
        return a - b;
      case "multiply":
        return a * b;
      case "divide":
        if (b === 0) throw new Error("Cannot divide by zero");
        return a / b;
      default:
        throw new Error("Unknown operation");
    }
  } catch (error) {
    console.error('Calculation error:', error);
    return `Error: ${error.message}`;
  }
}
```

**Original code:**
```
return "Error: Unknown operation";
```